static-site/src/util/parseMarkdown.js

```diff
@@ -0,0 +1,65 @@
+allowedAttributes.push("k", "k1", "k2", "k3", "k4", "kernelMatrix", "kernelUnitLength", "kerning", "keyPoints", "keySplines", "keyTimes");
+allowedAttributes.push("lang", "lengthAdjust", "letter-spacing", "lighting-color", "limitingConeAngle", "local");
+allowedAttributes.push("marker-end", "marker-mid", "marker-start", "markerHeight", "markerUnits", "markerWidth", "mask", "maskContentUnits", "maskUnits", "mathematical", "max", "media", "method", "min", "mode");
+allowedAttributes.push("name", "numOctaves");
+allowedAttributes.push("offset", "opacity", "operator", "order", "orient", "orientation", "origin", "overflow", "overline-position", "overline-thickness");
+allowedAttributes.push("panose-1", "paint-order", "path", "pathLength", "patternContentUnits", "patternTransform", "patternUnits", "ping", "pointer-events", "points", "pointsAtX", "pointsAtY", "pointsAtZ", "preserveAlpha", "preserveAspectRatio", "primitiveUnits");
+allowedAttributes.push("r", "radius", "referrerPolicy", "refX", "refY", "rel", "rendering-intent", "repeatCount", "repeatDur", "requiredExtensions", "requiredFeatures", "restart", "result", "rotate", "rx", "ry");
+allowedAttributes.push("scale", "seed", "shape-rendering", "slope", "spacing", "specularConstant", "specularExponent", "speed", "spreadMethod", "startOffset", "stdDeviation", "stemh", "stemv", "stitchTiles", "stop-color", "stop-opacity", "strikethrough-position", "strikethrough-thickness", "string", "stroke", "stroke-dasharray", "stroke-dashoffset", "stroke-linecap", "stroke-linejoin", "stroke-miterlimit", "stroke-opacity", "stroke-width", "surfaceScale", "systemLanguage");
+allowedAttributes.push("tabindex", "tableValues", "target", "targetX", "targetY", "text-anchor", "text-decoration", "text-rendering", "textLength", "to", "transform", "type");
+allowedAttributes.push("u1", "u2", "underline-position", "underline-thickness", "unicode", "unicode-bidi", "unicode-range", "units-per-em");
+allowedAttributes.push("v-alphabetic", "v-hanging", "v-ideographic", "v-mathematical", "values", "vector-effect", "version", "vert-adv-y", "vert-origin-x", "vert-origin-y", "viewBox", "viewTarget", "visibility");
+allowedAttributes.push("width", "widths", "word-spacing", "writing-mode");
+allowedAttributes.push("x", "x-height", "x1", "x2", "xChannelSelector");
+allowedAttributes.push("y", "y1", "y2", "yChannelSelector");
+allowedAttributes.push("z", "zoomAndPan");
+
+export const parseMarkdown = (mdString) => {
+  const sanitizerConfig = {
+    allowedTags,
+    allowedAttributes: {"*": allowedAttributes},
+    nonTextTags: ['style', 'script', 'textarea', 'option'],
+    transformTags: {
+      'a': function(tagName, attribs) { // eslint-disable-line
+        const url = new URL(attribs.href); // URL is not supported on Internet Explorer
+        if (url.hostname !== location.hostname) {
+          attribs.target = '_blank';
+          attribs.rel = 'noreferrer nofollow';
```

Comment:
@joverlee521 @jameshadfield I just got the following es-lint message in another context:
```
Using target="_blank" without rel="noopener noreferrer" is a security risk: see https://mathiasbynens.github.io/rel-noopenereslintreact/jsx-no-target-blank
```

Should we also have `noopener` here in addition to `noreferrer nofollow`?